apps/rush-lib/src/logic/PublishUtilities.ts

```diff
@@ -20,6 +20,13 @@ import { ChangeFiles } from './ChangeFiles';
 import { RushConfiguration } from '../api/RushConfiguration';
 import { DependencySpecifier, DependencySpecifierType } from './DependencySpecifier';
 import { Git, DEFAULT_GIT_TAG_SEPARATOR } from './Git';
+import { LockStepVersionPolicy } from '../api/VersionPolicy';
+import { SemVer } from 'semver';
+
+export interface IAllChanges {
+  packages: IChangeInfoHash;
```

Comment:
`pacakges` isn't the best name here. This is a dictionary of project names to their change data.

Can we change this type to a `Map<string, IChangeInfo>` and call this `changeInfoByProjectName`?

Reply:
what about `packageChanges`? i've seen that name in multiple functions like `sortChangeRequests`

Reply:
I've used `packageChanges` and `versionPolicyChanges` to be consistent with the rest of the codebase